src/components/settings/SafeModules/index.tsx

```diff
@@ -14,12 +17,22 @@ const NoModules = () => {
   )
 }
 
+const getModuleName = (chainId: string, address: string): string => {
+  if (sameAddress(getSpendingLimitModuleAddress(chainId), address)) {
+    return 'Spending limit module'
+  }
+
+  return 'Unknown module'
```

Comment:
Is this the same as in safe-react? I thought module names could also come in the known addresses.

Reply:
It seems that `safe-react` [doesn't use the known name](https://github.com/safe-global/safe-react/blob/dev/src/routes/safe/components/Settings/Advanced/ModulesTable.tsx#L74-L79). I've adjusted this to use the `module.name` if it exists and fallback to `getModuleName` if one doesn't exist. If an address book entry for said module exists, [that will be used](https://github.com/safe-global/web-core/blob/main/src/components/common/EthHashInfo/index.tsx#L87).